saleor/graphql/schema.graphql

```diff
@@ -5414,6 +5586,33 @@ type VerifyToken {
   accountErrors: [AccountError!]!
 }
 
+enum VolumeUnitsEnum {
+  US_G
+  MIL_US_G
+  US_QT
+  US_PINT
+  US_CUP
+  US_OZ
+  US_TBSP
+  US_TSP
+  CUBIC_MILLIMETER
+  CUBIC_CENTIMETER
+  CUBIC_DECIMETER
+  CUBIC_METER
+  L
+  CUBIC_FOOT
+  CUBIC_INCH
+  CUBIC_YARD
+  IMPERIAL_G
+  IMPERIAL_QT
+  IMPERIAL_PINT
+  IMPERIAL_OZ
+  IMPERIAL_TBSP
+  IMPERIAL_TSP
+  ACRE_IN
+  ACRE_FT
```

Comment:
@patrys  Do we need both? `ACRE` is not enough?

Reply:
Acre-inches and acre-feet are units of volume (consider a box with a base surface of an acre and the height of either an inch or a foot).

Reply:
Ok, so I'll leave both.